provider/doc_edits.go

```diff
@@ -21,3 +23,18 @@ var fixupEffectiveLabels = tfbridge.DocsEdit{
 		return content, nil
 	},
 }
+
+var secretsInPlainTextNoteRegexps = []*regexp.Regexp{
+	regexp.MustCompile(`~?> \*\*Warning:\*\* All arguments including the following potentially sensitive\nvalues will be stored in the raw state as plain text: .*\nRead more about sensitive data in state\.`),
+	regexp.MustCompile(`(?s)~?> \*\*((Warning)|(Note)):\*\* All arguments including .*as plain-text(\.)?( Read more about sensitive data in state\.)?`),
+}
+
+var removeSecretsInPlainTextNote = tfbridge.DocsEdit{
+	Path: "*",
+	Edit: func(path string, content []byte) ([]byte, error) {
+		for _, r := range secretsInPlainTextNoteRegexps {
+			content = r.ReplaceAllLiteral(content, []byte(""))
```

Comment:
Optional nit: `r.ReplaceAllLiteral` accepts a slice and operates based on length, so this is equivalent.
```suggestion
			content = r.ReplaceAllLiteral(content, nil)
```